simulator-samples/sample-swagger/src/test/java/org/citrusframework/simulator/OpenApiIT.java

```diff
@@ -0,0 +1,336 @@
+    @Autowired
+    @Qualifier("simulatorUiClient")
+    private HttpClient simulatorUiClient;
+
+    @CitrusTest
+    public void uiInfoShouldSucceed() {
+        $(http().client(simulatorUiClient)
+                .send()
+                .get("/api/manage/info")
+                .message()
+                .accept(MediaType.APPLICATION_JSON_VALUE)
+                .contentType(MediaType.APPLICATION_JSON_VALUE));
+
+        $(http().client(simulatorUiClient)
+                .receive()
+                .response(HttpStatus.OK)
+                .message()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .body("{" +
+                        "\"simulator\":" +
+                            "{" +
+                                "\"name\":\"REST Petstore Simulator\"," +
+                                "\"version\":\"@ignore@\"" +
+                            "}," +
+                            "\"activeProfiles\": []" +
+                        "}"));
+    }
+
+    @CitrusTest
+    public void addPetShouldSucceed() {
+        variable("name", "hasso");
+        variable("category", "dog");
+        variable("tags", "huge");
+        variable("status", "pending");
+
+        $(http().client(petstoreClientV3)
+                .send()
+                .post("/pet")
+                .message()
+                .accept(MediaType.APPLICATION_JSON_VALUE)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .body(new Resources.ClasspathResource("templates/pet.json")));
+
+        $(http().client(petstoreClientV3)
+                .receive()
+                .response(HttpStatus.OK));
+    }
+
+    @CitrusTest
+    public void addPetShouldFailOnMissingName() {
+        variable("category", "dog");
+        variable("tags", "huge");
+        variable("status", "pending");
+
+        $(http().client(petstoreClientV3)
+            .send()
+            .post("/pet")
+            .message()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body(new Resources.ClasspathResource("templates/pet_invalid.json")));
+
+        $(http().client(petstoreClientV3)
+            .receive()
+            .response(HttpStatus.INTERNAL_SERVER_ERROR));
+    }
+
+    @CitrusTest
+    public void deletePetShouldSucceed() {
+        variable("id", "citrus:randomNumber(10)");
+
+        $(http().client(petstoreClientV3)
+                .send()
+                .delete("/pet/${id}"));
+
+        $(http().client(petstoreClientV3)
+                .receive()
+                .response(HttpStatus.OK));
+    }
+
+    @CitrusTest
+    public void deletePetShouldFailOnWrongIdFormat() {
+
+        $(http().client(petstoreClientV3)
+            .send()
+            .delete("/pet/xxxx"));
+
+        $(http().client(petstoreClientV3)
+            .receive()
+            .response(HttpStatus.INTERNAL_SERVER_ERROR));
+    }
+
+//    @CitrusTest
```

Comment:
this is because..?

Reply:
I had a feature implemented that was able to access the api by alias. I had to remove this because i could not handle the registry, that managed the OpenApis due to this "unstable Citrus Instance" behaviour. This was a remnant using an alias.

Removed it.